includes/page-controller/class-wc-admin-page-controller.php

```diff
@@ -336,6 +336,12 @@ public function is_connected_page() {
 			$is_connected_page = isset( $current_page['js_page'] ) ? ! $current_page['js_page'] : true;
 		}
 
+		// Disable embed on the block editor.
+		$current_screen = get_current_screen();
```

Comment:
Just curious here, any reason why you'd make this assignment?

Reply:
Old habits die hard 🤷‍♂